UniversalWebshopScraper/generalized_scrapper/core/generalized_scrapper.py

```diff
@@ -514,7 +552,7 @@ def save_to_csv(self, save_path=None, category=None):
             website_name = self.shopping_website.replace("https://", "").replace("www.", "").split('.')[0]
 
             # Create directory for the specific website inside the 'scraped_data' folder
-            save_dir = os.path.join('scraped_data', website_name, category)
+            save_dir = os.path.join('../scraped_data', website_name, category)
```

Comment:
In general you never use this "../". If you need to go to a parallel folder, use project's main folder